docs/UML diagrams/Command.puml

```diff
@@ -0,0 +1,42 @@
+@startuml
+
```

Comment:
The icons for the UML diagrams for specifying method/attribute visibility don't conform to the module standards - this can be fixed by adding the directive `skinparam classAttributeIconSize 0`

![F9130137-0A1A-461F-A3C1-46E14A4E88DC](https://user-images.githubusercontent.com/18501113/197928845-0aa8adcf-c8b6-426c-b573-275bd81ebe75.jpeg)

Reply:
Class icons should be removed with the `hide circle` directive
![5C6844F7-96E9-4839-9750-0010EDD59D9E](https://user-images.githubusercontent.com/18501113/197929080-21cc16ef-eaac-40fc-8a41-b31afee313dd.jpeg)